users/views.py

```diff
@@ -189,3 +189,25 @@ def get(self, request, pk=None, format=None):
 
         serializer = self.get_serializer(page, many=True)
         return self.get_paginated_response(serializer.data)
+
+
+class ClaimPrizeResource(APIView):
+    def post(self, request, pk_student, pk_prize):
+        student = Student.objects.get(pk=pk_student)
+        prize = Prize.objects.get(pk=pk_prize)
```

Comment:
Since both `pk_student` and `pk_prize` are user-supplied values, we need to validate if these objects exist in our database